.vscode/settings.json

```diff
@@ -1,12 +1,12 @@
 {
     "editor.formatOnSave": false,
-    "eslint.autoFixOnSave": true,
     "eslint.validate": [
         "javascript",
         "javascriptreact",
-        { "language": "typescript", "autoFix": true }
+        "typescript"
     ],
     "eslint.quiet": true,
+    "eslint.workingDirectories": ["./lib/msal-core", "./lib/msal-common", "./lib/msal-browser", "./lib/msal-node", "./lib/msal-angular"],
```

Comment:
Are these changes meant to be part of this PR?

Reply:
I can take it out. I just got tired of putting it in every time I wanted linting to work properly.

Reply:
Done